system/greenhouse-ccloud/templates/plutono-storage-pluginpreset.yaml

```
@@ -0,0 +1,20 @@
+{{- if .Values.pluginPreset.enabled }}
+apiVersion: greenhouse.sap/v1alpha1
+kind: PluginPreset
+metadata:
+  name: plutono
+  namespace: {{ .Release.Namespace }}
+spec:
+  clusterSelector:
+    matchLabels:
+      greenhouse.sap/cluster-presets-enabled: "true"
```

please don't go with the old label

```suggestion
      greenhouse.sap/pluginpreset: "true"
```